tests/extras/fastapi/test_fastapi.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Tests that a pipeline with the FastAPI app runs correctly."""
+
+import time
+
+import pytest
+import ray
+import requests
+
+from aineko import AbstractNode, DatasetConsumer, Runner
+
+
+class FastAPIChecker(AbstractNode):
```

Comment:
I know this is beyond scope for this PR, but it would be nice to have a standardized testing framework for the api endpoints. I found the following that we might want to look into, which can automate part of the endpoint testing generation based on the openapi json definition of our fastapi app: https://schemathesis.readthedocs.io/en/stable/python.html

Reply:
Not aineko specific, but could be bundles as part of the fastapi node extra submodule